content/en/docs/collector/management.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Management
+weight: 23
+---
+
+Please be sure to review the following documentation:
+
+*  [Getting Started][otel-collector-getting-started] to understand how to
+   install the OpenTelemetry collector.
+*  [Configuration][otel-collector-configuration] for how to configure the 
+   OpenTelemetry collector, setting up telemetry pipelines.
+
+## Basics
+
+Telemetry collection at scale requires us to come up with a structured approach
+to manage agents. Typical agent management tasks (ordered from most common to
+least common) include:
```

Comment:
```suggestion
to manage agents. Typical agent management tasks include:
```
I think people may have different opinions about how common the particular task is, so perhaps omit the part in parenthesis.